src/main/resources/templates/new_view_datamart.html

```diff
@@ -825,21 +825,35 @@ <h3 class="panel-title">Selected</h3>
                 "render": function(data, type, row, meta) {
                     var result = '';
                     if ((row[8] && row[8] === "Dimension") || row[3] === "None" || !row[3]) {
-                        if (data && data.length > 0 ) {
+                        if (data && data[0].length > 0 ) {
```

Comment:
Shouldn't this be if (data && data.length > 0 && data[0].length > 0)? Because I think if data.length is 0, then accessing data[0] would give an error